pkg/sdn/api/v1beta3/conversion_test.go

```diff
@@ -0,0 +1,26 @@
+package v1beta3
```

Comment:
Done, added conversion tests to validate NetID field in the NetNamespace resource.